packages/react/src/components/drop/DropPod.tsx

```diff
@@ -93,20 +96,20 @@ const RDropPod: React.FunctionComponent<IRDropPodProps> = ({
 
     React.useEffect(() => {
         const element = document.createElement('div');
-        const portalRoot = document.querySelector(selector ?? Defaults.DROP_ROOT);
+        const portalRoot = document.querySelector(selector ?? DEFAULT_ROOT);

```

Comment:
Why remove the usage of `Defaults.DROP_ROOT`? I'm not sure this is a good idea

Reply:
Defaults.DROP_ROOT can be set, let's say in ADUI to change the default container of the drop element. With your new way it's not possible anymore

Reply:
I see :thinking:

Reply:
What Germain said, for example as you can see [here](https://github.com/coveo/admin-ui/blob/master/app/src/CoveoAdmin.ts#L211) in the admin-ui we want all the drops to render in the `#admin-drop-container` div and not the body.